lib/utils/testem-events.js

```diff
@@ -202,6 +212,8 @@ class TestemEvents {
         ui.writeLine(`\nExecution was recorded at ${testExecutionPath}`);
       }
 
+      ui.writeLine(`${browsersStarted.size} out of ${browserCount} browser(s) has ran tests.`);
```

Comment:
is this also recorded in the metadata file? it maybe useful information... 
i'd also consider re-wording the message to clarify `browserStarted.size` is the number of browsers that was launched.

Reply:
The # of browsers launched is not recored in the module metadata file. I thought that module metadata file should only be containing module related information. What about your thoughts?

Reply:
I think both the #requested and the #actually started would be very useful information. Knowing them would explain odd behaviors.

Reply:
@dcombslinkedin @step2yeung yes I agree that the data is very useful :) do you guys think that the data should be included in either test-execution-file or test-module-metadata? or it's fine to log it out? I will merge this pr after resolving this :)

Reply:
I'd personally like it in the metadata file. Logging it out doesn't help for things like CI runs.

Reply:
+1 on the metadata file, since I separate the usages of the two files as:
 - test-execution file for functionally rerunning the execution
 - test-module-metadata for informational data that helps with investigations

Reply:
Thank you for the feedback! I've made changes to add the data to the metadata file.
```
{
  "requested": "1 browser(s)",
  "launched": "1 browser(s)",
  "modules": [
    {
      ...
    },
    {
```

Reply:
awesome! Thanks, Chohee!